README.md

```diff
@@ -406,6 +406,20 @@ Once the above variables are set, run the following:
 make e2e-test
 ```
 
+Evn vars can be set to control tests specific timeouts:
+
+| Env var                                  | Description                                                                             | Default value |
+|------------------------------------------|-----------------------------------------------------------------------------------------|---------------|
+| E2E_TEST_RETRYINTERVAL                   | Retry interval for general operations                                                   | `10s`         |
+| E2E_TEST_TIMEOUTSHORT                    | Short timeout                                                                           | `1m`          |
+| E2E_TEST_TIMEOUTMEDIUM                   | Medium timeout: for readiness checks (e.g., ClusterServiceVersion, DataScienceCluster). | `7m`          |
+| E2E_TEST_TIMEOUTLONG                     | Long timeout: for more complex readiness (e.g., DSCInitialization, KServe).             | `10m`         |
+| E2E_TEST_GOMEGAEVENTUALLYTIMEOUT         | Timeout used for Eventually; overrides Gomega's default of 1 second.                    | `5m`          |
+| E2E_TEST_GOMEGAEVENTUALLYPOLLINTERVAL    | Polling interval for Eventually; overrides Gomega's default of 10 milliseconds.         | `2s`          |
+| E2E_TEST_GOMEGACONSISTENTLYTIMEOUT       | Duration used for Consistently; overrides Gomega's default of 2 seconds.                | `10s`         |
+| E2E_TEST_GOMEGACONSISTENTLYPOLLINTERVAL  | Polling interval for Consistently; overrides Gomega's default of 50 milliseconds.       | `2s`          |
+| E2E_TEST_FLAGS                           | Additional flags that can be passed to e2e-tests (see flags table below)                |               |
+
```

Comment:
From reviewing the code, it appears that all e2e flags are now configurable via environment variables. It would be helpful to add some documentation.

Reply:
All of the timeouts are related to `Gomega`, so I don't think we need to differentiate their names. 
We can also simplify things by removing the `retryInterval` and using the `gomegaEventuallyPollInterval` instead.

I propose we define the following variables:
- Default `Eventually` timeout and polling interval
- Default `Consistently` timeout and polling interval (we can reuse the polling interval from `Eventually`, but it's worth confirming)
- Medium and long timeouts for `Eventually`, along with corresponding polling intervals
- Medium and long timeouts for `Consistently`, along with corresponding polling intervals (even if not immediately needed, they could be useful in the future)

Reply:
Everything looks good to me. So far, we’ve defined the following:

- `defaultEventuallyTimeout`, `mediumEventuallyTimeout` and `longEventuallyTimeout`
- `defaultEventuallyPollInterval`
- `defaultConsistentlyTimeout`
-  `defaultConsistentlyPollInterval`

We could also consider adding a few more, but I’m unsure if we’ll need them. Open to everyone’s input:
- `mediumConsistentlyTimeout` and `longConsistentlyTimeout` for longer `Consistently` timeouts
- `mediumEventuallyPollInterval` and `longEventuallyPollInterval` for different `Eventually` polling intervals
- `mediumConsistentlyPollInterval` and `longConsistentlyPollInterval` for different `Consistently` polling intervals